components/AlumniCarousel.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @next/next/no-img-element */
+import { ALUMNI } from "../data/alumni";
+import { Navigation } from "swiper";
+import { SwiperSlide } from "swiper/react";
+import Carousel from "./Carousel";
+import "swiper/css";
+import "swiper/css/navigation";
+
+const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[37.5rem] shadow-2xl md:w-4/5 swiper-button-white";
+const ALUMNI_TILE =
+  "bg-[#7055FD] w-[20rem] h-[37.5rem] rounded-lg items-center p-5 flex flex-col shadow-2xl md:w-full md:flex-row";
+const ALUMNI_PROFILE = "flex flex-col items-center mx-auto";
+
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const AlumniTile = ({ name, file, position, prevRole, testimonial }) => {
+  return (
+    <div className={ALUMNI_TILE}>
+      <div className={ALUMNI_PROFILE}>
+        <img src={`/profiles/${file}`} alt="Profile Picture" className="w-50 mb-3 mt-5 rounded-full md:w-64 mx-auto" />
+        <h2 className="text-[2rem] font-bold">{name}</h2>
+        <h2 className="text-[1.25rem] font-semibold text-[#BAFBE4]">{position}</h2>
+        <h2 className="text-[1.25rem] font-medium mb-5">Previously: {prevRole}</h2>
+      </div>
+      <p className="text-[1.3rem] font-light italic leading-7 mx-auto md:mb-24 md:text-[1.5rem]">
```

Comment:
Can we make this bigger on desktop. Feels like there is a lot of white space. At least on high resolutions

Reply:
Gave it a shot. Take a look